### ISSUE TITLE
Can I build tensorboard with Protobuf 2.6.1?

### ISSUE THREAD

**Issue by Vogen** [dmlc/tensorboard#47]

**Comment by zihaolucky** [comment]
Nope, I think you have to install the version greater than the one in requirement, or it could be wrong while running bazel. Or you might want to install via PyPI then you can walk-around this issue.

**Comment by zihaolucky** [comment]
@Vogen New Protobuf could be easily installed, this one might help: https://github.com/zihaolucky/tensorboard-distro/blob/master/.travis.yml#L121-L156

**Comment by Vogen** [comment]
Thanks a lot!~
